assemblies/assembly-ba-artificial-intelligence.adoc

```diff
@@ -0,0 +1,57 @@
+A major transportation company has a fleet of trains and a network of train tracks. The company uses machine learning to identify geographic clusters where trains reported some type of problem. If a train reports a problem when it passes a specific position on the track it is recorded as data and if enough problems are recorded in the same location, you can identify a cluster. But that information is not complete enough to be useful so you can add digital decision to filter the data. For example, you can say that a failure must be a specific type of failure and must be reported by at least three trains. If that condition it met, you can use maths optimization to schedule a repair crew to investigate the track. Or if only one train has the same type of problem more than once at the location, you can use maths optimization to schedule the train for repair.
+
+As this example shows, you can combine machine learning to extract information from data analysis, digital decisioning to model the human knowledge and experience, and mathematical optimization to solve complex problems. These technologies use four industry standards:
+
+* Business Process Model and Notation (BPMN2)
++
+The BPMN2 specification is an Object Management Group (OMG) specification that defines standards for graphically representing a business process, defines execution semantics for the elements, and provides process definitions in XML format. BPMN2 can model computer and human tasks.
+
+* Case Management Model and Notation (CMMN)
++
+CMMN is used to model work methods that include various activities that might be performed in an unpredictable order depending on circumstances. CMMN models are event centered. CMMN expands overcomes limitations of what can be modeled with BPMN2 by supporting less structured work tasks and tasks driven by humans. By combining BPMN and CMMN you can create much more powerful models.
+
+* Decision Model and Notation (DMN)
++
+Decision Model and Notation (DMN) is a standard established by the OMG for describing and modeling operational decisions. DMN defines an XML schema that enables DMN models to be shared between DMN-compliant platforms and across organizations so that business analysts and business rules developers can collaborate in designing and implementing DMN decision services. The DMN standard is similar to and can be used together with the Business Process Model and Notation (BPMN) standard for designing and modeling business processes.
+
+* Predictive Model Markup Language (PMML)
++
+PMML is the language used to represent predictive models, mathematical models that use statistical techniques to uncover, or learn, patterns in large volumes of data. Predictive models use the patterns that they learn to predict the existence of patterns in new data. With PMML, you can share predictive models between applications. This data is exported as PMML files that can be consumed by DMN models. Therefore, DMN and PMML integrate well.
```

Comment:
Here it is important to add a note that major Machine Learning framework can persist the trained model AS a pmml file. So that, instead of just saving the trained model in a proprietary format, it could also be saved in an open standard format, such as PMML.